resources/monitoring/templates/prometheus/prometheus.yaml

```diff
@@ -250,7 +257,7 @@ spec:
 {{- end }}
 {{- if .Values.global.imagePullSecrets }}
   imagePullSecrets:
-{{ toYaml .Values.global.imagePullSecrets | indent 4 }}
+{{ include "kube-prometheus-stack.imagePullSecrets" . | trim | indent 4 }}
```

Comment:
Please check if this change is correct ? Not sure `kube-prometheus-stack.imagePullSecrets` if this path exists

Reply:
Yes, it's a new helper func. Also `global.imagePullSecrets` is not in use and empty in our values